hadoop-ozone/dev-support/checks/_lib.sh

```diff
@@ -72,3 +72,63 @@ create_aws_dir() {
     popd
   fi
 }
+
+
+# Function to find project paths for a given test class
+# Returns an array of project paths in PROJECT_PATHS
+find_project_paths_for_test_class() {
+  local test_class="$1"
+  # Reset the global PROJECT_PATHS array
+  PROJECT_PATHS=()
```

Comment:
I would change the script to print results to stdout and debug messages to stderr, instead of using a global variable.